src/app/(test)/ye/page.tsx

```diff
@@ -1,82 +1,30 @@
 'use client';
 
-import { Modal } from '@/components/Modals';
-import CheckedFileList from '@/components/common/CheckedFileList';
-import { useModal } from '@/hooks/useModal';
+import Button from '@/components/Button/Button';
+import FilterChip from '@/components/Chips/FilterChip';
+import { auth } from '@/firebase/firebaseClient';
+import { signInWithGithub } from '@/utils/signInWithGitHub';
+import { useEffect, useState } from 'react';
 
-const checkedData = [
-  {
-    fileName: 'Report_2024_08_19dddddsdhaskjdlkaj.pdf',
-    subTitle: 'Quarterly Financial Report',
-    createdAt: new Date(2024, 7, 19, 10, 30),
-  },
-  {
-    fileName: 'Presentation_2024_08_18.pptx',
-    subTitle: 'Project Kickoff Meeting',
-    createdAt: new Date(2024, 7, 18, 15, 45),
-  },
-  {
-    fileName: 'Notes_2024_08_17.docx',
-    subTitle: 'Meeting Notes',
-    createdAt: new Date(2024, 7, 17, 9, 0),
-  },
-  {
-    fileName: 'Invoice_2024_08_16.pdf',
-    subTitle: 'Client Invoice August',
-    createdAt: new Date(2024, 7, 16, 14, 20),
-  },
-  {
-    fileName: 'Agenda_2024_08_15.docx',
-    subTitle: 'Weekly Team Meeting',
-    createdAt: new Date(2024, 7, 15, 11, 0),
-  },
-  {
-    fileName: 'Agenda_2024_08_15.docx',
-    subTitle: 'Weekly Team Meeting',
-    createdAt: new Date(2024, 7, 15, 11, 0),
-  },
-];
+const options = ['option1', 'optixdasdasokd;laon2'];
 
 export default function TestPage() {
-  const [isModalOpen, handleClickTrigger] = useModal();
+  const [isLogin, setIsLogin] = useState(false);
 
-  return (
-    <div className="h-[2020rem]">
-      <button onClick={handleClickTrigger} type="button">
-        모달 띄우는 창
-      </button>
+  useEffect(() => {
+    const unsubscribe = auth.onAuthStateChanged(user => {
+      setIsLogin(!!user);
+    });
+
+    return () => unsubscribe();
+  }, []);
 
-      <Modal
-        gap={24}
-        padding={32}
-        hasShadow
-        hasDimmed
-        setIsModalOpen={handleClickTrigger}
-        isOpen={isModalOpen}
-      >
-        <Modal.Title size="sm">모달테스트 중 입니다.</Modal.Title>
-        <CheckedFileList checkedData={checkedData} />
-        <Modal.Text subtitle={['뭐라 적으실?', '두줄일경우']} helpText="그밑에 들어가는 텍스트" />
-        <Modal.Button
-          buttonText={{ left: 'left', right: 'right' }}
-          variant="doubleButton"
-          onClick={{
-            left: () => {
-              console.log('왼쪽');
-            },
-            right: () => {
-              console.log('오른쪽');
-            },
-          }}
-        />
-        {/* <Modal.Button
-            buttonText="button"
-            variant="singleButton"
-            onClick={() => {
-              console.log('gi');
-            }}
-          /> */}
-      </Modal>
-    </div>
+  return (
+    <>
+      <Button onClick={signInWithGithub}>로그인 버튼</Button>
+      <Button onClick={() => auth.signOut()}>로그아웃 버튼</Button>
```

Comment:
P5: (테스트 페이지이지만...!!!!) 로그아웃 버튼 사용할 때 이벤트 한 줄짜리라도 밖으로 빼기로 했던 것 같아서요! 다음에 사용할 때는 밖으로 빼서 사용하면 좋을 것 같습니다.

```
 const handleSignOut = async () => {
    try {
      await auth.signOut();
      setIsLogin(false); 
    } catch (error) {
      console.error('로그아웃 중 오류 발생', error);
    }
  };
  
```

```suggestion
      <Button onClick={handleSignOut}>로그아웃 버튼</Button>
```